### DISCUSSION TITLE
Fable.Remoting style integration with HTMX

### DISCUSSION THREAD

**Discussion by pkese** [Lanayx/Oxpecker#20] [Ideas]
A really powerful thing in F# world would be some sort of a seamless integration of HTMX in a similar manner as what  [Fable-remoting](https://github.com/Zaid-Ajaj/Fable.Remoting) does to REST APIs.

E.g. a way to generate and load snippets of HTMX without having to explicitly giving each snippet its own route.

A developer would write HTML templating code and calls local other functions rendering HTML from it, but when running inside of browser, those functions would automatically be exposed as http endpoints with its own routes.

**Comment by Lanayx** [comment]
Sounds like very exciting idea, however, I'm not sure about how to implement that, so first step might be to add Oxpecker integration to Fable-remoting repository.
